src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -595,9 +595,9 @@ def do_corpus_pruning(context, last_execution_failed, revision):
     return tasks_host.do_corpus_pruning(context, last_execution_failed,
                                         revision)
 
-  build_manager.setup_build(revision=revision)
+  build_setup_result = build_manager.setup_build(revision=revision)
   build_directory = environment.get_value('BUILD_DIR')
-  if not build_directory:
+  if not build_setup_result or not build_directory:
```

Comment:
do we even need to check build_directory ?

Reply:
Let me try removing it.